r/R/dplyr-summarize.R

```diff
@@ -166,14 +166,37 @@ format_aggregation <- function(x) {
 # appropriate combination of (1) aggregations (possibly temporary) and
 # (2) post-aggregation transformations (mutate)
 # The function returns nothing: it assigns into the `ctx` environment
-summarize_eval <- function(name, quosure, ctx, hash, recurse = FALSE) {
+summarize_eval <- function(name, quosure, ctx, hash) {
```

Comment:
`recurse` was not used anywhere in this function, so you cleaned it up from here, yeah?